skyportal/handlers/api/observation_plan.py

```diff
@@ -1603,21 +1603,32 @@ def observation_simsurvey(
 
         order = hp.nside2order(localization.nside)
         t = rasterize(localization.table, order)
-        result = t['PROB'], t['DISTMU'], t['DISTSIGMA'], t['DISTNORM']
-        hp_data = hp.reorder(result, 'NESTED', 'RING')
-        map_struct = {}
-        map_struct['prob'] = hp_data[0]
-        map_struct['distmu'] = hp_data[1]
-        map_struct['distsigma'] = hp_data[2]
-
-        distmean, diststd = parameters_to_marginal_moments(
-            map_struct['prob'], map_struct['distmu'], map_struct['distsigma']
-        )
 
-        distance_lower = astropy.coordinates.Distance(
-            np.max([1, (distmean - 5 * diststd)]) * u.Mpc
-        )
-        distance_upper = astropy.coordinates.Distance((distmean + 5 * diststd) * u.Mpc)
+        if 'DISTMU' in t:
+            result = t['PROB'], t['DISTMU'], t['DISTSIGMA'], t['DISTNORM']
+            hp_data = hp.reorder(result, 'NESTED', 'RING')
+            map_struct = {}
+            map_struct['prob'] = hp_data[0]
+            map_struct['distmu'] = hp_data[1]
+            map_struct['distsigma'] = hp_data[2]
+
+            distmean, diststd = parameters_to_marginal_moments(
+                map_struct['prob'], map_struct['distmu'], map_struct['distsigma']
+            )
+
+            distance_lower = astropy.coordinates.Distance(
+                np.max([1, (distmean - 5 * diststd)]) * u.Mpc
+            )
+            distance_upper = astropy.coordinates.Distance(
+                (distmean + 5 * diststd) * u.Mpc
```

Comment:
I know it probably never happens, but in principle you could get a `distance_upper` that is less than 1 as well...

Reply:
Adding a max here too :)